Tests/PushNotificationsNetworkableTests.swift

```diff
@@ -73,27 +76,25 @@ class PushNotificationsNetworkableTests: XCTestCase {
     }
 
     func testUnsubscribeAll() {
-        let exp = expectation(description: "It should successfully unsubscribe from all the interests")
-        self.networkService.unsubscribeAll {
+        let url = URL(string: "https://\(instanceId).pushnotifications.pusher.com/device_api/v1/instances/\(instanceId)/devices/apns/\(deviceId)/interests")!
+        let networkService = NetworkService(url: url, session: URLSession.shared)
+        networkService.unsubscribeAll {
             XCTAssert(true)
-            exp.fulfill()
-        }
-
-        waitForExpectations(timeout: 1) { error in
-            if let error = error {
-                XCTFail("waitForExpectations errored: \(error)")
-            }
         }
     }
 
     func testTrack() {
+        let url = URL(string: "https://\(instanceId).pushnotifications.pusher.com/reporting_api/v1/instances/\(instanceId)/events")!
+        let networkService = NetworkService(url: url, session: URLSession.shared)
         let userInfo = ["aps": ["alert": "hello"]]
-        self.networkService.track(userInfo: userInfo, eventType: ReportEventType.Delivery.rawValue, deviceId: "abc")
-        XCTAssert(true)
+        networkService.track(userInfo: userInfo, eventType: ReportEventType.Delivery.rawValue, deviceId: "abc")
+        XCTAssert(true) // Add integration test.
```

Comment:
This one doesn't do much, really. We would need to test if the value `abc` exists in the database.